.github/workflows/azure-static-web-apps-jolly-grass-05beb181e.yml

```diff
@@ -1,56 +1,56 @@
-
-on:
-  push:
-    branches:
-    - master
-  pull_request:
-    types: [opened, synchronize, reopened, closed]
-    branches:
-    - master
-
-jobs:
-  build_and_deploy_job:
-    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
-    runs-on: ubuntu-latest
-    name: Build and Deploy Job
-    steps:
-    - uses: actions/checkout@v1
-
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
-    - run: cd functions && npm install && npm run build
-
-    - uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: '3.1.201' # SDK Version to use.
-    - name: Build
-      run: dotnet run --configuration Release
-
-    - name: Build And Deploy
-      if: github.event.pull_request.head.repo.fork != true
-      id: builddeploy
-      uses: Azure/static-web-apps-deploy@v0.0.1-preview
-      with:
-        azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_JOLLY_GRASS_05BEB181E }}
-        repo_token: ${{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
-        action: 'upload'
-        ###### Repository/Build Configurations - These values can be configured to match you app requirements. ######
-        app_location: 'output' # App source code path
-        api_location: 'output/api' # Api source code path - optional
-        app_artifact_location: '' # Built app content directory - optional
-        ###### End of Repository/Build Configurations ######
-
-  close_pull_request_job:
-    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.head.repo.fork != true
-    runs-on: ubuntu-latest
-    name: Close Pull Request Job
-    steps:
-    - name: Close Pull Request
-      id: closepullrequest
-      uses: Azure/static-web-apps-deploy@v0.0.1-preview
-      with:
-        azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_JOLLY_GRASS_05BEB181E }}
-        action: 'close'
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+    - master
+
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - run: cd functions && npm install && npm run build
+
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '3.1.201' # SDK Version to use.
+    - name: Build
+      run: dotnet run --configuration Release
+
+    - name: Build And Deploy
+      if: github.event.pull_request.head.repo.fork != true && github.event.pull_request.head.repo == true
```

Comment:
what is this condition checking?

Reply:
When this check took place, then on the idea a bot with a preview page should be called.
But the build fell, as I understood from the fact that it was mainly made for fork repositories, and for those who make pull requests not from the fork, the build does not work, it skips (skips) this step because the build falls. Examples in Pull Requests below.

Reply:
And why the build crashes and the page with the example is not called up I cannot answer.